src/js/map/boundary_type_box.js

```diff
@@ -0,0 +1,161 @@
+import {Observable} from "../utils";
+
+let selectElement = $('.map-geo-select')[0];
+let selectedOption = null;
+let optionWrapper = null;
+let optionItem = null;
+let cookieName = 'boundary_selection';
```

Comment:
this is a constant and constants should be uppercase: `COOKIE_NAME` although cookie name is not very good. maybe `BOUNDARY_MODAL_COOKIE_NAME`

Reply:
done